aat/src/aat/java/uk/gov/hmcts/ccd/definitionstore/tests/functional/elasticsearch/ElasticsearchImportDefinitionTest.java

```diff
@@ -64,7 +70,29 @@ void shouldImportValidDefinitionMultipleTimes() {
             .when()
             .post("/import");
 
-        verifyIndexAndFieldMappings(true);
+        verifyIndexAndFieldMappingsWithRetries(true);
+    }
+
+    private void verifyIndexAndFieldMappingsWithRetries(boolean verifyNewFields) {
+        int retry = 1;
+        while (retry <= RETRY_COUNT) {
+            try {
+                verifyIndexAndFieldMappings(verifyNewFields);
```

Comment:
Take a look at this test util library. 
https://github.com/awaitility/awaitility/wiki/Usage#example-7---java-8

Reply:
wonderful sateesh, thanks for that. Good utility.